services/jira/jira-issue.service.js

```diff
@@ -1,100 +1,92 @@
 'use strict'
 
-const LegacyService = require('../legacy-service')
-const { makeBadgeData: getBadgeData } = require('../../lib/badge-data')
-const serverSecrets = require('../../lib/server-secrets')
+const Joi = require('joi')
+const BaseJiraService = require('./jira-base')
 
-module.exports = class JiraIssue extends LegacyService {
-  static get category() {
-    return 'issue-tracking'
+const schema = Joi.object({
+  fields: Joi.object({
+    status: Joi.object({
+      name: Joi.string().required(),
+      statusCategory: Joi.object({
+        colorName: Joi.string().required(),
+      }),
+    }).required(),
+  }).required(),
+}).required()
+
+module.exports = class JiraIssue extends BaseJiraService {
+  static render({ issue, statusName, color }) {
+    return {
+      label: issue,
+      message: statusName,
+      color,
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { color: 'lightgrey', label: 'jira' }
   }
 
   static get route() {
     return {
       base: 'jira/issue',
+      pattern: ':protocol(https?)/:host/:path?/:issueKey',
     }
   }
 
   static get examples() {
     return [
       {
         title: 'JIRA issue',
-        pattern: ':protocol/:hostAndPath+/:issueKey',
+        pattern: ':protocol/:host/:issueKey',
```

Comment:
Since `host` and `path` are broken out above, probably it makes sense to do the same here.